pulsar-functions/java-examples/pom.xml

```diff
@@ -41,6 +41,12 @@
       <artifactId>pulsar-functions-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+
```

Comment:
Can we automatically include the schema from functions-api?

Reply:
done. I add schema-api as part of functions-api